proto/documents/v1alpha/web_publishing.proto

```diff
@@ -0,0 +1,330 @@
+syntax = "proto3";
+
+package com.mintter.documents.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "mintter/backend/genproto/documents/v1alpha;documents";
+
+// Service exposed by the local application backend for
+// configuring and managing remote Mintter Sites.
+service WebPublishing {
+  // Adds a site configuration to the local app backend.
+  rpc AddSite(AddSiteRequest) returns (SiteConfig);
+
+  // Removes site configuration from the local app backend.
+  rpc RemoveSite(RemoveSiteRequest) returns (google.protobuf.Empty);
+
+  // Lists configured sites.
+  rpc ListSites(ListSitesRequest) returns (ListSitesResponse);
+
+  // Lists web publication records for a given document among the configured web sites.
+  rpc ListWebPublicationRecords(ListWebPublicationRecordsRequest) returns (ListWebPublicationRecordsResponse);
+}
+
+// Request to add a site.
+message AddSiteRequest {
+  // Required. Site hostname.
+  string hostname = 1;
+
+  // Optional. Invite token for the site. Not needed
+  // if the site already knows our Account ID.
+  string invite_token = 2;
```

Comment:
Since there is also a remote `RedeemInviteToken` which actually takes a token and associates the bearer's account ID with the role in that token on a table in the remote site, how precisely the `invite_token` in this local `AddSite` call is suppose to work? will the backend make a call to the `RedeemInviteToken` under the hood before returning from the `AddSite` call (if it had a token in it)? will the frontend call `RedeemInviteToken` before `AddSite`? cc @horacioh @ericvicenti @burdiyan

Reply:
I think you and @burdiyan can decide on this. Happy to do it from the UI if you think that makes sense, but also fine if it happens inside the `AddSite` call.

Reply:
@juligasa Also thought about it before, and I assumed the backend would call RedeemToken with this token value indeed.